### DISCUSSION TITLE
SPC (CONHAZO & PTOTSVRTSTM)

### DISCUSSION THREAD

**Discussion by grfritz** [weather-gov/api#378] [Q&A]
We currently retrieve the SPC CONHAZO (Convective Outlook) and PTOTSVRTSTM (Probability of Severe Thunderstorm) elements from the NDFD for use in our Skywarn scripts.  The NDFD has its own set of occasional issues, including sluggish response.   Please advise whether these SPC elements are available via the API and if not, whether they might be included in a future update.  Greatly appreciate any feedback.

**Comment by StephenClouse** [comment]
We do not currently serve any SPC data via API.

Personally it's something I'd like to offer but it is not currently part of our roadmap.
